packages/cli/src/modules/insights/__tests__/insights-compaction.service.test.ts

```diff
@@ -320,7 +322,13 @@ if (dbType === 'sqlite' && !globalConfig.database.sqlite.poolSize) {
 				jest.useFakeTimers();
 				try {
 					// ARRANGE
-					const insightsCompactionService = Container.get(InsightsCompactionService);
+					const insightsCompactionService = new InsightsCompactionService(
+						mock<InsightsByPeriodRepository>(),
+						mock<InsightsRawRepository>(),
+						mock<InsightsConfig>({
+							compactionIntervalMinutes: 60,
+						}),
+					);
```

Comment:
Is this change necessary?

Reply:
At first I thought the transaction locked issue was caused by this test (which starts the scheduler on the real timer), that's why I simplified it. 
I still think it makes sense to keep it, cause it's a bit overkill to run the full service just to test if a scheduling works